docker-compose.yml

```diff
@@ -57,6 +56,9 @@ services:
       # This should correspond to the server declared in PHPStorm `Preferences | Languages & Frameworks | PHP | Servers`
       # Then PHPStorm will use the corresponding path mappings
       PHP_IDE_CONFIG: serverName=localhost
+      CORS_ALLOW_ORIGIN: "*"
+      API_DOMAIN: ".*"
+      SHARED_COOKIE_DOMAIN: ""
```

Comment:
I don't understand why you chose these values.
- CORS_ALLOW_ORIGIN could be empty now, couldn't it? There are no more cross-origin requests after this PR.
- API_DOMAIN is a regex now, which makes no sense. If this is really needed (I assume you did this for accessing the API from a mobile phone), could you use another environment variable such as `ADDITIONAL_TRUSTED_HOSTS` or `TRUSTED_API_HOSTS` or something similar? That way it would become clearer that the API is not running on a regex domain.
- If we follow this PR up with #3262, then SHARED_COOKIE_DOMAIN can be removed

Reply:
> I don't understand why you chose these values.
> 
>     * CORS_ALLOW_ORIGIN could be empty now, couldn't it? There are no more cross-origin requests after this PR.
> 
Maybe, i did not test it. But framework.yml adds localhost and and caddy as CORS Hosts, maybe it does not work if you don't include the same host explicitly.
The goal of this restructuring is that you can access the instance on your local machine from your phone in the same network. (And but with lower priority an instance running on github codespaces or gitpod)
>     * API_DOMAIN is a regex now, which makes no sense. If this is really needed (I assume you did this for accessing the API from a mobile phone), could you use another environment variable such as `ADDITIONAL_TRUSTED_HOSTS` or `TRUSTED_API_HOSTS` or something similar? That way it would become clearer that the API is not running on a regex domain.
I can do that
> 
>     * If we follow this PR up with [.helm: move services to subpaths #3262](https://github.com/ecamp/ecamp3/pull/3262), then SHARED_COOKIE_DOMAIN can be removed

SHARED_COOKIE_DOMAIN is removed in https://github.com/ecamp/ecamp3/pull/3262/files#diff-09c8c391e3a627b701028c8c82756440688ff60ec3fe6664b93f4a92e68e5df0

Reply:
Tested empty CORS_ALLOW_ORIGIN and removed it in https://github.com/ecamp/ecamp3/pull/3222/commits/cf9d20fd1805a3fdb745f50b635b72f2570c8529
Renamed API_DOMAIN in https://github.com/ecamp/ecamp3/pull/3222/commits/dc65e65dcc5095a84b08c5570c76af5178ec845e